### ISSUE TITLE
How to convert GNSS coordinates from 'lat, lon' to the pitch coordinates'? 

### ISSUE THREAD

**Issue by mcjeong95** [AtomScott/SoccerTrack#33]
Hi, I've just downloaded SoccerTrack dataset and try to explore GNSS data.

However, the coordinates of GNSS data are in 'lat, lon', so I couldn't compare the coordinates which were obtained from camera.

**How can I convert GNSS coordinates from 'lat, lon' to the pitch coordinates?**

Thanks for releasing valuable soccer dataset publicly!

**Comment by AtomScott** [comment]
Hi!
Thanks for the question.

You need to calculate the homography transform and project the coordinates.

Here's a file containing the corresponding points between standard pitch coordinates (in meters) and Lat-Lon coordinates. 

https://drive.google.com/file/d/1tnYiwtSjdLf1J8qXjvN2A0_ZBGBIwEvl/view?usp=drivesdk

Btw, I'm guessing you are one of the authors from the MLSA paper:

https://dtai.cs.kuleuven.be/events/MLSA22/papers/MLSA22_paper_8096.pdf

GPS-OTS alignment is something we thought would be useful in alleviating GT annotation cost for soccertrack so it would be great to see the results in 11 vs 11 setting.

**Comment by mcjeong95** [comment]
Yes, as you guessed I'm one of the authors from the MLSA paper, and we're trying to apply our algorithm to SoccerTrack.

If it works well, I'll let you know :)

Thanks for all your help, Scott!
